docs/adrs/0003-unsubscribing-and-replying.md

```diff
@@ -0,0 +1,48 @@
+# Unsubscribing and Replying
+
+Date: 2023-09-14
+
+## Status
+
+Accepted
+
+## Context
+
+Message recipients must legally be allowed to remove consent (referred to as unsubscribing) 
+for both email and SMS messages. To honor unsubscribe actions, this application stores consent status 
+in order to filter out messages when appropriate. 
+
+## Decision
+
+Messages will always be sent unless there is a relevant `unsubscribed` record.
+
+Each unsubscribe or subscribe action is recorded along with the phone number or email address it came with. 
+No other information (such as a name or ID number) is stored with this consent. 
+
+
+### Email
+Unsubscribing from emails is done by following a link, which is added by default in a message at the 
```

Comment:
The architectural decision here was to use Mailgun's unsubscribe feature where they provide a URL template variable substitution feature and they host the endpoint for unsubscribes.